notebook/static/base/js/i18n.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// Module to handle i18n ( Internationalization ) and translated UI
+
+define([
+    'jed',
+    'moment',
+    'json!../../../i18n/nbjs.json',
+    'json!../../../i18n/de/LC_MESSAGES/nbjs.json',
```

Comment:
Maybe it's possible to use require in a way that it doesn't fail if it can't find a module? It looks like we can get the language code from `navigator` before we call `define()`, so we could stick it into a URL - but we don't know at that point if we're asking for a supported language.

I think I'd like to get this worked out before we merge the translation machinery - sorry for another delay, but loading the messages seems pretty fundamental, and the current mechanism is one we know we'll need to replace.

The `json!` thing is pretty neat, though - I didn't know about that before.